srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -28,6 +28,9 @@
     private final static String ALBUMIN_SUCCESS = "ALBUMIN^3.0^02/02/2015@14:35:12^g/dl";
     private final static String INVALID_LAB = "This is invalid. ~@!#$";
     private final static String VALID_LAB_NO_UNITS = "ALBUMIN^3.0^02/02/2015@14:35:12^";
+    private final static String VALID_HEALTH_FACTORS = "08/25/2014^REFUSED INFLUENZA IMMUNIZATION\n" +
```

Comment:
The idea was to mirror exactly what we would get back from calling the RPC so that it could be handled appropriately.